sapl/sessao/views.py

```diff
@@ -161,17 +162,18 @@ def verifica_votacoes_abertas(request):
     return True
 
 
-def verifica_sessao_iniciada(request, spk):
+def verifica_sessao_iniciada(request, spk, is_leitura=False):
     logger = logging.getLogger(__name__)
     sessao = SessaoPlenaria.objects.get(id=spk)
 
     if not sessao.iniciada or sessao.finalizada:
         username = request.user.username
-        logger.info('user=' + username + '. Não é possível abrir matérias para votação. '
-                    'Esta SessaoPlenaria (id={}) não foi iniciada ou está finalizada.'.format(spk))
-        msg = _('Não é possível abrir matérias para votação. '
+        aux_text = 'leitura' if is_leitura else 'votação'
+        logger.info('user=' + username + '. Não é possível abrir matérias para {}. '
+                    'Esta SessaoPlenaria (id={}) não foi iniciada ou está finalizada.'.format(aux_text, spk))
```

Comment:
Vamos aproveitar e consertar essa mistura de concatenação e interpolação de strings? :) 

```suggestion
        logger.info('user={}. Não é possível abrir matérias para {}. '
                    'Esta SessaoPlenaria (id={}) não foi iniciada ou está finalizada.'.format(username, aux_text, spk))
```

Reply:
Corrigido